configuration/redis/metadata.yaml

```diff
@@ -3,7 +3,7 @@ schemaVersion: v1
 type: configuration
 name: redis
 version: v1
-status: alpha
+status: stable
```

Comment:
Is the configuration API already marked as Stable?

Reply:
It will be stable in this release.